.github/workflows/ci_cd.yml

```diff
@@ -336,15 +351,38 @@ jobs:
           docker load --input /tmp/ingestion_server/ingestion_server.tar
 
       - name: Build docs
-        run: just sphinx-make
+        run: |
+          just sphinx-make
+          sudo chown "$USER:$USER" -R ./api
+          mv ./api/build/html /tmp/preview
+
+      # Otherwise we end up with any extra stuff not git-ignored in the gh-pages branch
+      - name: Recreate working directory to avoid superfluous files
+        run: |
+          cd ..
+          sudo rm -rf openverse-api
+          mkdir openverse-api
+
+      - uses: actions/checkout@v3
+        with:
+          ref: gh-pages
+
+      - name: Merge preview with existing gh-pages
+        run: |
+          cd ..
+          cp -r ./openverse-api /tmp/gh-pages
+          # trash the existing preview folder and replace it with the newly generated one
+          # if the preview hasn't been pushed yet this will still exit(0)
+          rm -rf /tmp/gh-pages/_preview/${{ github.event.pull_request.number }}
```

Comment:
I think this also needs `sudo`, unfortunately.

Reply:
https://github.com/WordPress/openverse-api/runs/7617377580?check_suite_focus=true#step:10:293

Reply:
Thanks, fix is here: https://github.com/WordPress/openverse-api/pull/841